### PR TITLE
(PC-7348) spike scrollview modal 

### PR DESCRIPTION
**Pull request by emmajmn**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-7348

## Checklist

I have:

- [ ] Made sure the title of my PR follows the convention `[PC-XXX] <summary>`.
- [ ] Made sure my feature is working on the relevant real / virtual devices.
- [ ] Written **unit tests** for my feature.
- [ ] Added new reusable components to the AppComponents page and communicate to the team on slack
- [ ] Added a **screenshot** for UI tickets.
- [ ] Attached a **ticket number** for any added TODO/FIXME \
       (for tech tasks, give the best context about the TODO resolution: what/who/when).

## Screenshots

| \*iOS - [Phone name] | \*Android - [Phone name] | Tablet - [Phone name] | Little - [Phone name] |
| -------------------- | :----------------------: | --------------------: | --------------------: |
|                      |                          |                       |                       |

## Deploy hard

If native code (ios/android) was modified, **after** the PR is merged, on the master branch, upgrade the **app version** (+1 patch):

- if you want an hard deployment of the testing environment, use `yarn trigger:testing:deploy`

### CODE REVIEW

**Review by antoinewg — APPROVED** [review]
Joli !

File: src/features/offer/pages/tests/__snapshots__/Offer.test.tsx.snap [hunk 1]
```diff
@@ -1927,2229 +1928,2248 @@ Tous les détails du film sur AlloCiné: http://www.allocine.fr/film/fichefilm_g
...
-                                                    "fontSize": 15,
-                                                    "lineHeight": 20,
+                                                    "color": "#b6c1cd",
+                                                    "fontFamily": "System",
+                                                    "fontSize": 13,
+                                                    "fontWeight": undefined,
+                                                    "marginBottom": 7,
+                                                    "marginTop": 2,
+                                                    "textAlign": "center",
+                                                    "width": 32,
```
Comment (justinemignot) [inline src/features/offer/pages/tests/__snapshots__/Offer.test.tsx.snap:2228]: c'est normal ce snapshot qui change autant ? :( c'est dur à lire

File: src/ui/components/modals/AppModal.tsx [hunk 2]
```diff
@@ -28,8 +29,11 @@ export const AppModal: FunctionComponent<Props> = ({
   onRightIconPress,
   children,
   titleNumberOfLines,
+  isScrollable,
```
Comment (justinemignot) [inline src/ui/components/modals/AppModal.tsx:32]: tu peux mettre "= false" par defaut

**Review by justinemignot — COMMENTED** [review]
Je pense qu'il faut ajouter des marges sous le boutons "Choisir les options" et "Confirmer la réservation" non ?

File: src/ui/components/modals/AppModal.tsx (hunk 2 above)
Comment (emmajmn) [reply src/ui/components/modals/AppModal.tsx:32]: 👍🏻

File: src/features/offer/pages/tests/__snapshots__/Offer.test.tsx.snap (hunk 1 above)
Comment (emmajmn) [reply src/features/offer/pages/tests/__snapshots__/Offer.test.tsx.snap:2228]: yes j'ai relu en gros il ajoute la scrollview autour de la vue qui contient le content de la modale du coup ça fait des décalages ^^'

**Pull request merged**